### PR TITLE
chore(ci): abort build on connection errors

### LINKED ISSUE #4383: Improve build information for Kubernetes connection errors
**Description**

Currently for Kubernetes connection errors the build fails and a new build is scheduled. However, it is currently not immediately visible why the build failed.

This issues seeks to improve that by:
* Marking the build as aborted
* Setting a build description that explains why it failed

### PR DESCRIPTION
**Pull request by pihme**
## Description

If a connection issue is detected:
- the build will be set to aborted
- the description will state the reason for the abort

## Related issues

closes #4383

## Pull Request Checklist

- [ X ] All commit messages match our [commit message guidelines](https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines)
- [ X ] The submitting code follows our [code style](https://github.com/zeebe-io/zeebe/wiki/Code-Style)
- [ X ] If submitting code, please run `mvn clean install -DskipTests` locally before committing

### CODE REVIEW

**Comment by pihme** [comment]
This is what it looks like now
image

File: Jenkinsfile [hunk 1]
```diff
@@ -200,6 +200,9 @@ pipeline {
             // Retrigger the build if there were connection issues
             script {
                 if (connectionProblem()) {
+                    currentBuild.result = 'ABORTED'
+                    currentBuild.description = "Aborted due to connetion error"
```
Comment (menski) [inline Jenkinsfile:204]:
```suggestion
                    currentBuild.description = "Aborted due to connection error"
```

**Review by menski — APPROVED** [review]
Looks good, just fix the typo before merging. :shipit:

**Pull request merged**